GooglePlayInstant/Editor/PlayInstantPublisher.cs

```diff
@@ -48,16 +48,27 @@ public static void Build()
                 return;
             }
 
+            ZipApk(baseApkPath, zipFilePath);
+        }
+
+        /// <summary>
+        /// Creates a ZIP file containing the specified APK
+        /// </summary>
+        public static void ZipApk(string apkPath, string zipFilePath)
```

Comment:
I will be able to reuse this too if you use the following signature:
`CreateZipFile(string inputDirectoryName, string inputFileName, string zipFilePath)`

Reply:
Resolved